src/notes/FluxNotesEditor.jsx

```diff
@@ -1316,10 +1365,10 @@ class FluxNotesEditor extends React.Component {
                 }
                 remainder = remainder.substring(start + trigger.trigger.length);
 
-                // FIXME: Temporary work around that adds spaces when needed to @-phrases inserted via mic
-                if (start !== 0 && trigger.trigger.startsWith('@') && !before.endsWith(' ')) {
-                    transform = this.insertPlainText(transform, ' ');
-                }
+                // // FIXME: Temporary work around that adds spaces when needed to @-phrases inserted via mic
```

Comment:
Maybe remove this?

Reply:
Spoke to Dylan about this. We want to leave it in since this was originally in here for dictation. Might need to revisit it in the future so leaving in the commented out code